pages/common/stow.md

```diff
@@ -0,0 +1,24 @@
+# stow
+
+> Symlink manager.
+> Often used to manage dotfiles.
+
+- Symlink all files recursively to {{/path/to/target/directory}}:
```

Comment:
Please could you remove the starting `/` from the path here?

Reply:
Oh, actually, maybe it should also be `path/to/target_directory` or just `path/to/directory`. Up to you 😄 👍